package.json

```diff
@@ -16,37 +16,34 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/SuperSaaS/n8n-nodes-supersaas.git"
+    "url": "git+https://github.com/SuperSaaS/n8n-nodes-supersaas.git"
   },
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
     "build": "tsc && gulp build:icons",
     "dev": "tsc --watch",
     "format": "prettier nodes credentials --write",
-    "lint": "eslint nodes/**/*.ts nodes/**/*.json package.json",
-    "lintfix": "eslint nodes/**/*.ts nodes/**/*.json package.json --fix",
-    "prepublishOnly": "npm run build && npm run lint -c .eslintrc.prepublish.js nodes credentials package.json"
```

Comment:
.eslintrc. - this is deprecated, although there is a migration tool, it just did so many warnings and errors that it is faster just to read the documentation on eslint.config.mjs rather than try figuring out the tool